packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -506,4 +496,80 @@ describe('Kamino strategy creation SDK Tests', () => {
+
+    let lowerPriceBpsDifference = new Decimal(10.0);
+    let upperPriceBpsDifference = new Decimal(11.0);
+
+    let buildNewStrategyIxs = await kamino.getBuildStrategyIxns(
+      'ORCA',
+      new Decimal(0.0001),
+      newStrategy.publicKey,
+      newPosition.publicKey,
+      signer.publicKey,
+      new Decimal(PricePercentage.discriminator),
+      [lowerPriceBpsDifference, upperPriceBpsDifference],
+      new PublicKey('EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v'),
+      new PublicKey('USDH1SM1ojwWUga67PGrgFWUHibbjqMvuMaDkRJTgkX')
+    );
+
+    let ixs: TransactionInstruction[] = [];
+    ixs.push(createRaydiumStrategyAccountIx);
+    ixs.push(buildNewStrategyIxs[0]);
+    const createStratTx = await kamino.getTransactionV2Message(signer.publicKey, ixs);
+    const createStratTransactionV0 = new VersionedTransaction(createStratTx);
+    createStratTransactionV0.sign([newStrategy, signer]);
+    //@ts-ignore
+    let txHash = await sendAndConfirmTransaction(kamino._connection, createStratTransactionV0);
+    console.log('create strategy tx hash', txHash);
+
+    let strategySetupIxs: TransactionInstruction[] = [];
+    buildNewStrategyIxs[1].slice(0, 4).map((ix) => strategySetupIxs.push(ix));
+    const setupStratTx = await kamino.getTransactionV2Message(signer.publicKey, strategySetupIxs);
+    const setupStratTransactionV0 = new VersionedTransaction(setupStratTx);
+    setupStratTransactionV0.sign([signer]);
+
+    //@ts-ignore
+    txHash = await sendAndConfirmTransaction(kamino._connection, setupStratTransactionV0);
+    console.log('setup strategy tx hash', txHash);
+
+    let strategySetupFeesIxs: TransactionInstruction[] = [];
+    buildNewStrategyIxs[1].slice(4).map((ix) => strategySetupFeesIxs.push(ix));
+    strategySetupFeesIxs.push(buildNewStrategyIxs[2]);
+    const setupStratFeesTx = await kamino.getTransactionV2Message(signer.publicKey, strategySetupFeesIxs);
+    const setupStratFeesTransactionV0 = new VersionedTransaction(setupStratFeesTx);
+    setupStratFeesTransactionV0.sign([signer]);
+    //@ts-ignore
+    txHash = await sendAndConfirmTransaction(kamino._connection, setupStratFeesTransactionV0);
+    console.log('setup strategy fees tx hash', txHash);
+
+    // verify strategy rebalance params
+    let strategyData = await kamino.getStrategies([newStrategy.publicKey]);
+    expect(strategyData[0]?.rebalanceRaw[0] == lowerPriceBpsDifference);
+    expect(strategyData[0]?.rebalanceRaw[2] == upperPriceBpsDifference);
+
+    // open position
+    const openPositionIxn = buildNewStrategyIxs[3];
+    const openPositionMessage = await kamino.getTransactionV2Message(signer.publicKey, [openPositionIxn]);
+    const openPositionTx = new VersionedTransaction(openPositionMessage);
+    openPositionTx.sign([signer, newPosition]);
+
+    //@ts-ignore
+    const openPositionTxId = await sendAndConfirmTransaction(kamino._connection, openPositionTx);
+    console.log('openPositionTxId', openPositionTxId);
```

Comment:
overall the new code is good. i think it's missing asserts (such that we lock in the functionality), a bit too much boilerplate and a bit of the structure of the return types. if you're going to split the txns by same index always (like `buildNewStrategyIxs[3]`) then you might as well return an object and destructure it:

```ts
const { openPositionIxn, strategySetupFeesIxs, ...} = buildNewStrategyIxs;
```